File: Modul 4/Soal 1/PRAK401-2410817210014-ZAVIER GHANI.py
def main():
    data = input("Masukkan bilangan kelipatan dan simbol (dipisahkan spasi): ").split()
    kelipatan = int(data[0])
    simbol = data[1]

    hasil = []
    for i in range(1, 51):
        if i % kelipatan == 0:
            hasil.append(simbol)
        else:
            hasil.append(str(i))

    print(" ".join(hasil))

main()

File: Modul 4/Soal 1/PRAK401-2410817210014-ZAVIERGHANI.c
#include <stdio.h>

int main() {
    int kelipatan;
    char simbol[2];

    printf("Masukkan bilangan kelipatan dan simbol (dipisahkan spasi): ");
    scanf("%d %s", &kelipatan, simbol);

    for (int i = 1; i <= 50; i++) {
        if (i % kelipatan == 0) {
            printf("%s ", simbol);
        } else {
            printf("%d ", i);
        }

        if (i % 10 == 0) {
        }
    }

    return 0;
}
